.pre-commit-config.yaml

```
@@ -31,11 +31,7 @@ repos:
       hooks:
           - id: pyupgrade
             args: [--py37-plus]
-    - repo: local
+    - repo: https://github.com/sphinx-contrib/sphinx-lint
+      rev: v0.9.1
       hooks:
-          - id: rst
-            name: rst
-            entry: rst-lint --encoding utf-8
-            files: ^(CHANGES.rst|README.rst)$
-            language: python
-            additional_dependencies: [pygments, restructuredtext_lint]
+        - id: sphinx-lint
```

I recommend this too, it looks for single backticks in RST, which should usually be double backticks for code formatting (MD uses single).
```suggestion
        - id: sphinx-lint
          args: [--enable=default-role]
```